src/main/docker/Dockerfile

```diff
@@ -14,40 +14,52 @@
 # limitations under the License.
 #
 
+# Preliminary image that includes Elasticsearch and related files
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.3.2 AS builder
+
 #
-# Preliminary image that downloads and extracts Elasticsearch
+# The actual image definition
 #
-FROM cafapi/opensuse-jre8:1 AS builder
+FROM cafapi/opensuse-jre8:1
 
-RUN zypper -n install which hostname
-RUN zypper -n install unzip
+ENV ELASTIC_CONTAINER=true
 
-# Download and Extract Elasticsearch
-ADD https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-6.3.2.zip /opt/
+COPY --from=builder /opt/jdk-10.0.2/ /opt/jdk-10.0.2/
+ENV JAVA_HOME=/opt/jdk-10.0.2
```

Comment:
Do we have to set this environment variable?  I think it would be better if we just set it in the [start.sh](https://github.com/CAFapi/opensuse-elasticsearch6-image/blob/develop/src/main/docker/scripts/start.sh) file so that it only effects Elasticsearch and not also the classification worker.

Reply:
JAVA_HOME is not used in start script (docker-entrypoint.sh), or it is not global when it is set in start script?

Reply:
I don't understand what you're saying here.  My original point was that setting it globally like this will have an impact on more than just Elasticsearch.